pkg_resources/__init__.py

```diff
@@ -43,11 +44,7 @@
 import importlib
 from pkgutil import get_importer
 
-try:
-    import _imp
-except ImportError:
-    # Python 3.2 compatibility
-    import imp as _imp
+import _imp
```

Comment:
I'm only assuming by the comment. since the minimal supported python version is now 3.8, this should be fine right?

Reply:
Done in #4096 if you wanna keep it separate